posts/2021-04-16-mp-context-propagration-21004.adoc

```diff
@@ -0,0 +1,353 @@
+seo-description: In 21.0.0.4, we now offer Admin Center, MicroProfile Context Propagration 1.2, LDAP connection support for Kerberos authentication and a number of enhancements (expandLocation property for AutoExpand, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, Automatic Cleanup of Leaked Connections, Single Sign-On (SSO) LTPA and JWT support authentication filter and Single Sign-On (SSO) LTPA and JWT support authentication filter)
+blog_description: In 21.0.0.4, we now offer Admin Center, MicroProfile Context Propagration 1.2, LDAP connection support for Kerberos authentication and a number of enhancements (expandLocation property for AutoExpand, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, Automatic Cleanup of Leaked Connections, Single Sign-On (SSO) LTPA and JWT support authentication filter and Single Sign-On (SSO) LTPA and JWT support authentication filter)
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Admin Center, MicroProfile Context Propagration 1.2, LDAP connection support for Kerberos and a number of enhancements in Open Liberty 21.0.0.4
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+// tag::intro[]
+
+In 21.0.0.4, Admin Center 1.0 joins Open Liberty, providing a web-based GUI to help configure, manage, and monitor your running server, along with support for viewing and interacting with Java Batch jobs and OpenID Connect (OIDC) client and token management. MicroProfile Context Propagation is updated to version 1.2, which addresses the difference in how link:{url-prefix}/blog/2021/03/31/microprofile-config-2.0.html[MicroProfile Config 2.0] treats empty value configuration properties. LDAP connection now supports Kerberos authentication. Also present in this release are a number of enhancements: expandLocation property for AutoExpand, the ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, automatic cleanup of leaked connections, Single Sign-On (SSO) LTPA and JWT support authentication filter, and Single Sign-On (SSO) LTPA and JWT support authentication filter
+
+
+In link:{url-about}[Open Liberty] 21.0.0.4:
+
+* <<admincenter, Admin Center 1.0>>
+* <<mpContextPropagation, MicroProfile Context Propagation 1.2>>
+* <<ldapconnection, LDAP connection supports Kerberos authentication>>
+* <<expandLocation, expandLocation property for AutoExpand>>
+* <<requestTiming, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature>>
+* <<JDBC, Automatic Cleanup of Leaked Connections>>
+* <<SSO, Single Sign-On (SSO) LTPA and JWT support authentication filter>>
+* <<bugs, Notable bugs fixed in this release>>
```

Comment:
Change the order and titles of the blog to be:
* <<admincenter, Admin Center 1.0>>
* <<mpContextPropagation, MicroProfile Context Propagation 1.2>>
* <<ldapconnection, LDAP connection supports Kerberos authentication>>
* <<SSO, Single Sign-On (SSO) LTPA and JWT support authentication filter>>
* <<requestTiming, Control thread dump collection for hung requests in the Request Timing feature>>
* <<JDBC, Automatic cleanup of leaked connections>>
* <<expandLocation, Specify location for expanded applications>>
* <<bugs, Notable bugs fixed in this release>>


For each title change, follow up with the owner to make sure they're ok with change and make sure the title of each section below matches the above.

Reply:
Done

Reply:
Verified title changes with authors of requestTiming && expandLocation: Both have no issues